debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -290,10 +299,18 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
                     + "In '" + SnapshotIsolationMode.READ_UNCOMMITTED.getValue()
                     + "' mode neither table nor row-level locks are acquired, but connector does not guarantee snapshot consistency.");
 
+    public static final Field TASK_DATABASE_NAMES = Field.create("task.database.dbnames")
+            .withDisplayName("Task Database Names")
+            .withType(Type.STRING)
+            .withWidth(Width.MEDIUM)
+            .withImportance(Importance.HIGH)
+            .withValidation(Field::isRequired)
+            .withDescription("The name of the databases assigned to a task");
+
     private static final ConfigDefinition CONFIG_DEFINITION = HistorizedRelationalDatabaseConnectorConfig.CONFIG_DEFINITION.edit()
             .name("SQL Server")
             .type(
-                    DATABASE_NAME,
```

Comment:
We should keep the old one for the sake of compatibility. A validation failure should be raised if _both_ are given at once.

Reply:
Filed as https://github.com/sugarcrm/debezium/issues/47.